kairon/__main__.py

```diff
@@ -21,8 +33,12 @@ def main():
     connect(host=Utility.environment['database']['url'])
     logger.info(arguments.bot)
     logger.info(arguments.user)
-    start_training(arguments.bot, arguments.user, arguments.token)
+    if arguments.which == 'training_parser':
```

Comment:
1. change from training_parser to training
2. also need to check model trianing